src/content/doc-surrealql/statements/create.mdx

```diff
@@ -21,7 +21,7 @@ CREATE [ ONLY ] @targets
 	[ CONTENT @value
 	  | SET @field = @value ...
 	]
-	[ RETURN NONE | RETURN BEFORE | RETURN AFTER | RETURN DIFF | RETURN @statement_param, ... ]
+	[ RETURN NONE | RETURN BEFORE | RETURN AFTER | RETURN DIFF | RETURN [ VALUE ] @statement_param, ... ]
```

Comment:
Thanks for the PR (and for noticing this behaviour in the first place)! I'm thinking that maybe this should be the way to state it at the end since RETURN VALUE only allows a single param.

----

RETURN @statement_param, .. | RETURN VALUE @statement_param

Reply:
Fully understand! Let me know if the changes are what you were thinking.

Reply:
I'm thinking the other way around! RETURN can be followed with @statement_param, ... because it can take multiple parameters, while RETURN VALUE can only take one @statement_param.

```surql
CREATE s SET name = 'Mr. Stuff' RETURN name, "other";
CREATE s SET name = 'Mr. Stuff' RETURN VALUE name;
```

Reply:
Makes so much more sense why those were there my bad!